spec/views/comments/_comment.html.erb_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
```

Comment:
This spec is unrelated to the feature introduced in this PR but still necessary to test the low-quality feature that I broke on my previous PR.